test/instantiation/test_instantiate_all_pipelines.py

```diff
@@ -299,11 +301,12 @@ def test_instantiate_PETSurfaceCrossSectional(cmdopt):
 # def test_instantiate_PETSurfaceLongitudinal():
 #     from os.path import dirname, join, abspath
 #     from clinica.pipelines.pet_surface.pet_surface_pipeline import PetSurface
+#     from clinica.utils.pet import Tracer
 #
 #     root = dirname(abspath(join(abspath(__file__), pardir)))
 #     root = join(root, 'data', 'PETSurfaceLongitudinal')
 #     parameters = {
-#         'acq_label': 'FDG',
+#         'acq_label': Tracer.FDG,
```

Comment:
Unrelated to this PR, but I was wondering why this test is commented?
If it fails because something is broken atm, we can add a skip decorator such that you have statistics on the number of skipped tests. WDYT?

Reply:
Likely a question for @mdiazmel, but indeed a skip marker may be more elegant.

Reply:
We would also be able to specify a reason for skipping each test which would clarify things and avoid having such questions down the road.

Reply:
Let's discuss this with him next meeting.

Reply:
Good point, thanks.